docs/pages/brownfield/lifecycle-listeners.mdx

```diff
@@ -0,0 +1,116 @@
+---
+title: Configuring lifecycle listeners
+sidebar_title: Lifecycle listeners
+description: Learn about the mechanism that allows the Expo Modules API to hook into the lifecycle of your app.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+import { PlatformTag } from '~/ui/components/Tag/PlatformTag';
+
+Some Expo libraries need to handle system events such as deep links, push notifications, and configuration changes by implementing `AppDelegate` or `Activity`/`Application` lifecycle callbacks.
```

Comment:
```suggestion
Some Expo libraries need to handle system events such as deep links, push notifications, and configuration changes by implementing `Activity`/`Application` or `AppDelegate`  lifecycle callbacks.
```

Let's follow the platform order as per our writing styleguide for smaller details/references, such as this one.